metric_providers/psu/energy/ac/xgboost/machine/provider.py

```diff
@@ -1,13 +1,7 @@
-import sys
 import os
 from io import StringIO
 import pandas
 
-CURRENT_DIR = os.path.dirname(os.path.abspath(__file__))
-sys.path.append(f"{CURRENT_DIR}/../../../../../../lib")
-sys.path.append(CURRENT_DIR)
-
-#pylint: disable=import-error, wrong-import-position
 import model.xgb as mlmodel
```

Comment:
I get an error here when I try to run the runner with the XGBoost model enabled:

```
>   import model.xgb as mlmodel
E   ModuleNotFoundError: No module named 'model'

../metric_providers/psu/energy/ac/xgboost/machine/provider.py:5: ModuleNotFoundError
```
The model referenced in this branch is on the same commit as the one in main: https://github.com/green-coding-berlin/spec-power-model/tree/95f13ad95ced02350c75841616306115a1402e93

Does the spec-power-model need an update for this too? I see this error both in tests, and using the runner.py manually.